RaisedGardenBeds/ModEntry.cs

```diff
@@ -12,8 +12,23 @@ namespace RaisedGardenBeds
 {
 	public class ModEntry : Mod
 	{
-		// common
-		internal static ModEntry Instance;
+
+        internal static readonly string GameContentAssetPath = Path.Combine("Mods", "blueberry.rgb.Assets");
+
+        internal static readonly string GameContentEndOfNightSpritesPath = Path.Combine(GameContentAssetPath, "EndOfNightSprites");
+        internal static readonly string GameContentEventDataPath = Path.Combine(GameContentAssetPath, "EventData");
+        internal static readonly string GameContentCommonTranslationDataPath = Path.Combine(GameContentAssetPath, "CommonTranslations");
+        internal static readonly string GameContentItemTranslationDataPath = Path.Combine(GameContentAssetPath, "ItemTranslations");
+
+        internal static readonly string LocalAssetPath = "assets";
+
+        internal static readonly string LocalEndOfNightSpritesPath = Path.Combine(LocalAssetPath, "endOfNightSprites.png");
+        internal static readonly string LocalEventDataPath = Path.Combine(LocalAssetPath, "eventData.json");
+
+        internal static readonly string ContentPackPath = Path.Combine(LocalAssetPath, "ContentPack");
+
+        // common
+        internal static ModEntry Instance;
```

Comment:
Did you mean to copy these fields from `AssetManager`? They appear to be defined in both files.